django/contrib/admin/static/admin/css/nav_sidebar.css

```diff
@@ -117,3 +117,16 @@
         max-width: 100%;
     }
 }
+
+#nav-filter {
+    width: 100%;
+    box-sizing: border-box;
+    padding: 5px 16px;
+    margin: 5px 0;
+    border: none;
+    outline: none;
```

Comment:
We shouldn't override outline - I know it's a bit ugly but keyboard users rely on it to know where they are when tabbing through the page - blinking cursor alone isn't enough.

Reply:
Good point!

Reply:
Done